### DISCUSSION TITLE
Problem after docker-compose.

### DISCUSSION THREAD

**Discussion by samsonpwnz** [jertel/vuegraf#125] [Q&A] [answered]
I am running vuegraf using the included docker-compose file. Everything seems to start correctly, my vuegraf.json is getting read as I can see in logs it's access emporia correctly. My issue is there is no dashboard in grafana. Even if I try to import the included dashboard it is just blank. Thanks for any help.

**Comment by samsonpwnz** [comment]
After some more digging, this might have something to do with my paths in the docker-compose.yaml
Attached is log from grafana, 
logger=local.finder t=2023-07-24T11:43:33.275931865Z level=warn msg="Skipping finding plugins as directory does not exist" path=/usr/share/grafana/plugins-bundled
logger=query_data t=2023-07-24T11:43:33.278557429Z level=info msg="Query Service initialization"
logger=live.push_http t=2023-07-24T11:43:33.280810592Z level=info msg="Live Push Gateway initialization"
logger=infra.usagestats.collector t=2023-07-24T11:43:34.900196941Z level=info msg="registering usage stat providers" usageStatsProvidersLen=2
logger=provisioning.datasources t=2023-07-24T11:43:34.90030495Z level=error msg="can't read datasource provisioning files from directory" path=/etc/grafana/provisioning/datasources error="open /etc/grafana/provisioning/datasources: no such file or directory"
logger=provisioning.plugins t=2023-07-24T11:43:34.900324783Z level=error msg="Failed to read plugin provisioning files from directory" path=/etc/grafana/provisioning/plugins error="open /etc/grafana/provisioning/plugins: no such file or directory"
logger=provisioning.notifiers t=2023-07-24T11:43:34.900337982Z level=error msg="Can't read alert notification provisioning files from directory" path=/etc/grafana/provisioning/notifiers error="open /etc/grafana/provisioning/notifiers: no such file or directory"
logger=provisioning.alerting t=2023-07-24T11:43:34.900357843Z level=error msg="can't read alerting provisioning files from directory" path=/etc/grafana/provisioning/alerting error="open /etc/grafana/provisioning/alerting: no such file or directory"
logger=provisioning.alerting t=2023-07-24T11:43:34.900368278Z level=info msg="starting to provision alerting"
logger=provisioning.alerting t=2023-07-24T11:43:34.90037672Z level=info msg="finished to provision alerting"

Still not sure how to set this up correctly.

**Comment by jertel** [comment, marked as answer]
Grafana support was dropped from the project a while ago. The included dashboard is for InfluxDB2.

**Reply by samsonpwnz** [reply]
If i go to 192.168.1.x:8086 I get "404 page not found"

**Reply by jertel** [reply]
I just looked at that docker compose template and I see it's still trying to start Grafana and using the old InfluxDB v1. Another community member contributed that compose yaml file years ago. I don't personally use it so it's become stale.

It will need some attention to bring it up to date.

**Reply by jertel** [reply]
I pushed some improvements to the compose template. It will pull and start Vuegraf and InfluxDB2. You will need to refer to the project readme for more information on additional DB setup and Vue account setup.

**Reply by samsonpwnz** [reply]
That worked wonders. After following the rest of the readme as to how to set up influx and user/token I try to import the dashboard and get this error. 
Failed to import dashboard: Error: resource_type="check" err="failed to apply resource" metadata_name="competent-hopper-f6f003" err_msg="failed to create checks[\"competent-hopper-f6f003\"]: Could not create task from check: loc 0:0-0:0: unknown operator \"<INVALID_OP>\"\n" metadata_name="noshing-borg-b6f003" err_msg="failed to create checks[\"noshing-borg-b6f003\"]: Could not create task from check: loc 0:0-0:0: unknown operator \"<INVALID_OP>\"\n"

**Reply by jertel** [reply]
Looks similar to what another user reported in #120. How are you attempting to import the dashboard?

**Reply by samsonpwnz** [reply]
I have tried the method in the readme,  which gets me "Error: failed to find inputs at path "/var/lib/influxdb2/influx_dashboard.json": stat /var/lib/influxdb2/influx_dashboard.json: no such file or directory"
I have also tried importing the file into influxdb from the web gui by dragging and dropping and also pasting the raw json into the provided window. All those methods give the same error.

**Reply by jertel** [reply]
You cannot use the GUI to import. It must be done by following the instructions. The reason for the "no such file or directory" error is because the location that you copied the dashboard json file isn't correct. The instructions state:

> You will need to apply this template file to your running InfluxDB instance. Copy the influx_dashboard.json file into your host's influxdb2 path. If you followed the Setup instructions above, the path would be /home/myuser/influxdb2. 

Which path did you copy the dashboard json file into?

**Reply by samsonpwnz** [reply]
I copied into /home/myusername/influxdb2 . 
I'm not very well versed in docker, I am a copy/pasta person so as you probably already figured out I have no clue how to troubleshoot this very well.

**Reply by jertel** [reply]
Ok, I will try to get you going. Copy the influx_dashboard.json file into the container as follows:

```
docker cp <path-to-vuegraf-project>/influx_dashboard.json influxdb:/var/lib/influxdb2/
```

Then try to import it via this command, using your token you saved when you setup InfluxDB2:

```
docker exec influxdb influx apply -f /var/lib/influxdb2/influx_dashboard.json --org vuegraf --force yes -t <my-influx-token>
```

**Reply by samsonpwnz** [reply]
That did it! Thank you very much for taking the time to help me out!
